File: WarehouseSystem/WarehouseSystem/ItemDisposalSaveDto.cs
using Newtonsoft.Json;

namespace WarehouseSystem
{
    class ItemDisposalSaveDto
    {
        public string disposalId { get; set; }
        public string itemPoId { get; set; }
        public string qty { get; set; }

        public ItemDisposalSaveDto (string disposalId,string itemPoId,string qty)
        {
            this.disposalId = disposalId;
            this.itemPoId = itemPoId;
            this.qty = qty;
        }

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}

File: warehouse-system/src/main/java/com/who/warehousesystem/dto/ItemDisposalSaveDto.java
package com.who.warehousesystem.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor

public class ItemDisposalSaveDto {

    private Integer disposalId;
    private Integer itemPoId;
    private Integer qty;
}
